diff --git a/packages/headless/src/modules/brand/useBrand.ts b/packages/headless/src/modules/brand/useBrand.ts
index 5a123aded9..0e9c620d08 100644
--- a/packages/headless/src/modules/brand/useBrand.ts
+++ b/packages/headless/src/modules/brand/useBrand.ts
@@ -161,7 +161,7 @@ export const useBrand = () => {
         "meta",
         "portal",
         "@context.oneTimePurchases"
-      ]) === "hidden"
+      ]) === "hide"
   );
 
   const i18nMessages = computed<BrandMeta["i18n"]>(() =>
