diff --git a/packages/modules-auth/src/components/Login.vue b/packages/modules-auth/src/components/Login.vue
--- a/packages/modules-auth/src/components/Login.vue
+++ b/packages/modules-auth/src/components/Login.vue
@@ -97,6 +97,7 @@
 import { Link, Markdown } from "@upmind/ui";
 import { computed, ref } from "vue";
 import { useI18n } from "vue-i18n";
+import { SESSION_TEMPLATES } from "@upmind-automation/client-vue";
 import { Hero } from "@upmind-automation/foundation";
 import { Back } from "@upmind-automation/foundation";
 import { Section } from "@upmind-automation/foundation";
@@ -173,7 +174,7 @@ const meta = computed(() => ({
 
 const { component: templateVariant } = useAuthTemplate(
   () => template.value,
-  () => props.templates
+  () => SESSION_TEMPLATES
 );
 const templateProps = computed(() => omit(props, ["templates"]));
 
