diff --git a/packages/client-vue/src/components/form/renderers/index.ts b/packages/client-vue/src/components/form/renderers/index.ts
index ce69b20..5d39e46 100644
--- a/packages/client-vue/src/components/form/renderers/index.ts
+++ b/packages/client-vue/src/components/form/renderers/index.ts
@@ -1,9 +1,12 @@
 import { registerEntry } from "@upmind/ui";
 import EnumToggleGroupRenderer from "./EnumToggleGroupRenderer.vue";
 import { tester as enumToggleGroupTest } from "./EnumToggleGroupRenderer.vue";
+import LookupRenderer from "./LookupRenderer.vue";
+import { tester as lookupTest } from "./LookupRenderer.vue";
 
 // -----------------------------------------------------------------------------
 
 export const formRenderers = [
+  registerEntry(LookupRenderer, lookupTest),
   registerEntry(EnumToggleGroupRenderer, enumToggleGroupTest)
 ];
