diff --git a/packages/modules-foundation/src/renderers/useFormRenderers.ts b/packages/modules-foundation/src/renderers/useFormRenderers.ts
index 1363926da3..c098a8b1c4 100644
--- a/packages/modules-foundation/src/renderers/useFormRenderers.ts
+++ b/packages/modules-foundation/src/renderers/useFormRenderers.ts
@@ -1,4 +1,5 @@
 import { hasInjectionContext, inject } from "vue";
+import "../../../modules-product/src";
 import type { FormRendererEntry, UseFormRenderers } from "./renderer.types";
 import type { InjectionKey } from "vue";
 
