diff --git a/packages/headless/src/modules/client-orders/client-orders.mappers.ts b/packages/headless/src/modules/client-orders/client-orders.mappers.ts
index b70b5f7350..ac8e381daa 100644
--- a/packages/headless/src/modules/client-orders/client-orders.mappers.ts
+++ b/packages/headless/src/modules/client-orders/client-orders.mappers.ts
@@ -71,7 +71,7 @@ export function mapOrderDetail(raw?: IOrder): ClientOrderDetail {
     cancellationReason: raw?.contract?.cancellation_reason ?? undefined,
     notes: raw?.notes,
     customFields: raw?.custom_fields,
-    contractId: raw?.contract_id,
+    contractId: raw?.id,
     brandId: raw?.brand_id,
     referrer:
       raw?.account?.affiliate_referral?.affiliate_account?.account?.client?.[0]
