diff --git a/packages/headless/src/modules/client-orders/useClientOrders.actions.ts b/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
--- a/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
+++ b/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
@@ -359,7 +359,7 @@
    */
   function setCriteria(intent: Partial<ClientOrdersQueryModel>): void {
     const next: Partial<ClientOrdersQueryModel> = omitBy(
-      { ...intent },
+      { ...intent, sort: undefined },
       isUndefined
     );
 
