diff --git a/packages/headless/src/modules/orders/useOrders.actions.ts b/packages/headless/src/modules/orders/useOrders.actions.ts
index 9bc303aa19..1d82698d17 100644
--- a/packages/headless/src/modules/orders/useOrders.actions.ts
+++ b/packages/headless/src/modules/orders/useOrders.actions.ts
@@ -356,7 +356,10 @@ export function createOrdersActions(
     const next: Partial<OrdersQueryModel> = omitBy({ ...intent }, isUndefined);
 
     if (intent.filters) {
-      const filters: OrdersFilterModel = { ...intent.filters };
+      const filters: OrdersFilterModel = {
+        ...(query.criteria.value.filters ?? {}),
+        ...intent.filters
+      };
       const statusBranch = filters["status.code"];
       if (statusBranch?.eq && statusBranch?.neq) {
         filters["status.code"] = { eq: statusBranch.eq };
