diff --git a/packages/headless/src/modules/orders/orders.schemas.ts b/packages/headless/src/modules/orders/orders.schemas.ts
index f017c15a51..f7611dd6e9 100644
--- a/packages/headless/src/modules/orders/orders.schemas.ts
+++ b/packages/headless/src/modules/orders/orders.schemas.ts
@@ -83,7 +83,6 @@ export function useQuerySchema(): QuerySchema {
           "status.code": {
             type: "object",
             additionalProperties: false,
-            maxProperties: 1,
             properties: {
               eq: {
                 type: ["array", "null"],
