diff --git a/packages/headless/src/modules/client-address/client-address.services.ts b/packages/headless/src/modules/client-address/client-address.services.ts
index 1708af336..a6a5b2257 100644
--- a/packages/headless/src/modules/client-address/client-address.services.ts
+++ b/packages/headless/src/modules/client-address/client-address.services.ts
@@ -475,13 +475,14 @@ async function setDefault(
   clientId: ClientAddressServices["clientId"],
   captureError: ClientAddressErrorCapture
 ): Promise<void> {
   const { t } = useI18n();
   const { put, useUrl } = useQuery();
 
-  if (!isAddressable(clientId.value)) {
+  const { isAuthenticated } = useActiveSession().useMeta();
+  if (!(isAuthenticated.value || !clientId.value)) {
     return Promise.reject(new NotAuthenticatedError());
   }
 
   return put<IAddress>({
     mutationKey: [...queryKey, addressId, "default"],
     url: useUrl(`clients/${clientId.value}/addresses/${addressId}`),
