diff --git a/packages/headless/src/modules/client-address/client-address.services.ts b/packages/headless/src/modules/client-address/client-address.services.ts
index 1708af336..35ce06ba5 100644
--- a/packages/headless/src/modules/client-address/client-address.services.ts
+++ b/packages/headless/src/modules/client-address/client-address.services.ts
@@ -445,13 +445,14 @@ async function remove(
   clientId: ClientAddressServices["clientId"],
   captureError: ClientAddressErrorCapture
 ): Promise<void> {
   const { t } = useI18n();
   const { del, useUrl } = useQuery();
 
-  if (!isAddressable(clientId.value)) {
+  const { isAuthenticated } = useActiveSession().useMeta();
+  if (!(isAuthenticated.value || !clientId.value)) {
     return Promise.reject(new NotAuthenticatedError());
   }
 
   return del<null>({
     mutationKey: [...queryKey, addressId, "remove"],
     url: useUrl(`clients/${clientId.value}/addresses/${addressId}`),
