diff --git a/packages/headless/src/modules/client-billing-settings/useBillingSettings.actions.ts b/packages/headless/src/modules/client-billing-settings/useBillingSettings.actions.ts
index 379ce0dd6..789858549 100644
--- a/packages/headless/src/modules/client-billing-settings/useBillingSettings.actions.ts
+++ b/packages/headless/src/modules/client-billing-settings/useBillingSettings.actions.ts
@@ -62,8 +62,8 @@ export function createBillingSettingsActions(
     }
 
     return new Promise<boolean>(resolve => {
-      const stop = watch(query.isFetched, fetched => {
-        if (!fetched) return;
+      const stop = watch(query.isSuccess, success => {
+        if (!success) return;
         stop();
         resolve(!query.error.value);
       });
