diff --git a/packages/headless/src/modules/session-store/useSession.meta.ts b/packages/headless/src/modules/session-store/useSession.meta.ts
index ad36cef46..0a19f9a9b 100644
--- a/packages/headless/src/modules/session-store/useSession.meta.ts
+++ b/packages/headless/src/modules/session-store/useSession.meta.ts
@@ -62,9 +62,9 @@ export function createSessionMeta(_sessionId?: string) {
   //   pushing `?.` onto every future consumer of the type.
   const hasDelegatedProducts = computed(
     () =>
-      !!activeUser.value?.delegatedIds?.[UpmindObjectTypes.CONTRACTS_PRODUCT]
+      !!activeUser.value.delegatedIds[UpmindObjectTypes.CONTRACTS_PRODUCT]
         ?.length ||
-      !!activeUser.value?.delegatedIds?.[UpmindObjectTypes.CLIENT]?.length
+      !!activeUser.value.delegatedIds[UpmindObjectTypes.CLIENT]?.length
   );
 
   const isUnverified = computed(() => {
