diff --git a/packages/headless/src/modules/session-store/useSession.meta.ts b/packages/headless/src/modules/session-store/useSession.meta.ts
index ad36cef46..06aba1017 100644
--- a/packages/headless/src/modules/session-store/useSession.meta.ts
+++ b/packages/headless/src/modules/session-store/useSession.meta.ts
@@ -63,8 +63,7 @@ export function createSessionMeta(_sessionId?: string) {
   const hasDelegatedProducts = computed(
     () =>
       !!activeUser.value?.delegatedIds?.[UpmindObjectTypes.CONTRACTS_PRODUCT]
-        ?.length ||
-      !!activeUser.value?.delegatedIds?.[UpmindObjectTypes.CLIENT]?.length
+        ?.length
   );
 
   const isUnverified = computed(() => {
