diff --git a/packages/headless/src/modules/client-orders/useClientOrders.actions.ts b/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
index d6ccd6b1ed..cfd8f962ef 100644
--- a/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
+++ b/packages/headless/src/modules/client-orders/useClientOrders.actions.ts
@@ -163,7 +163,7 @@ export function createClientOrdersActions(
       query.criteria.value.pagination?.offset
     ],
     ([isFetched, hasError, total, offset]) => {
-      if (!isFetched || hasError || total !== 0) return;
+      if (!isFetched || hasError) return;
       if (!offset) return;
       query.setCriteria({
         pagination: {
