diff --git a/packages/headless/src/modules/client-orders/client-orders.mappers.ts b/packages/headless/src/modules/client-orders/client-orders.mappers.ts
index b70b5f7350..66fc09d0d0 100644
--- a/packages/headless/src/modules/client-orders/client-orders.mappers.ts
+++ b/packages/headless/src/modules/client-orders/client-orders.mappers.ts
@@ -74,7 +74,7 @@ export function mapOrderDetail(raw?: IOrder): ClientOrderDetail {
     contractId: raw?.contract_id,
     brandId: raw?.brand_id,
     referrer:
-      raw?.account?.affiliate_referral?.affiliate_account?.account?.client?.[0]
+      raw!.account!.affiliate_referral!.affiliate_account!.account!.client![0]
   };
 }
 
