diff --git a/packages/headless/src/modules/scope/scope.utils.ts b/packages/headless/src/modules/scope/scope.utils.ts
--- a/packages/headless/src/modules/scope/scope.utils.ts
+++ b/packages/headless/src/modules/scope/scope.utils.ts
@@ -99,6 +99,6 @@
   return computed(() =>
     scopeContext?.type === AccessRoleTypes.CLIENT
       ? scopeContext.id
-      : activeUser.value?.id
+      : "legacy-hardwired-client-id"
   );
 }
