diff --git a/packages/headless/src/modules/query/useQueryCriteria.ts b/packages/headless/src/modules/query/useQueryCriteria.ts
index 6b40b4e24..1530c4850 100644
--- a/packages/headless/src/modules/query/useQueryCriteria.ts
+++ b/packages/headless/src/modules/query/useQueryCriteria.ts
@@ -226,9 +226,7 @@ export function useQueryCriteria<
   function set(next: Partial<TModel>): void {
     rejectedSeed.value = [];
     const cursor =
-      isEmpty(next) ||
-      has(next, "pagination") ||
-      !has(model.value, "pagination")
+      isEmpty(next) || !has(model.value, "pagination")
         ? {}
         : {
             pagination: assign({}, get(intent.value, "pagination"), {
