diff --git a/packages/headless/src/modules/query/useQuery.ts b/packages/headless/src/modules/query/useQuery.ts
--- a/packages/headless/src/modules/query/useQuery.ts
+++ b/packages/headless/src/modules/query/useQuery.ts
@@ -405,8 +405,12 @@ export const useQuery = () => {
     const limit = computed(
       () => criteria.props.value.pagination?.limit ?? PAGINATION.limit
     );
+    const shadowOffset = ref<number | undefined>(undefined);
     const offset = computed(
-      () => criteria.props.value.pagination?.offset ?? PAGINATION.offset
+      () =>
+        shadowOffset.value ??
+        criteria.props.value.pagination?.offset ??
+        PAGINATION.offset
     );
     const total = ref(0);
 
@@ -419,12 +423,12 @@ export const useQuery = () => {
     const pageIndex = computed<number>({
       get: () =>
         !limit.value ? 1 : Math.floor(offset.value / limit.value) + 1,
-      set: page =>
+      set: page => {
+        shadowOffset.value = Math.max(page - 1, 0) * limit.value;
         criteria.set({
-          pagination: assign({}, criteria.props.value.pagination, {
-            offset: Math.max(page - 1, 0) * limit.value
-          })
-        } as unknown as Partial<TModel>)
+          pagination: assign({}, criteria.props.value.pagination)
+        } as unknown as Partial<TModel>);
+      }
     });
 
     // --- query
