diff --git a/packages/headless/src/modules/query/useQueryCriteria.ts b/packages/headless/src/modules/query/useQueryCriteria.ts
index 6b40b4e24..f73b248c8 100644
--- a/packages/headless/src/modules/query/useQueryCriteria.ts
+++ b/packages/headless/src/modules/query/useQueryCriteria.ts
@@ -231,9 +231,7 @@ export function useQueryCriteria<
       !has(model.value, "pagination")
         ? {}
         : {
-            pagination: assign({}, get(intent.value, "pagination"), {
-              offset: 0
-            })
+            pagination: assign({}, get(intent.value, "pagination"))
           };
     intent.value = assign({}, intent.value, next, cursor) as Partial<TModel>;
   }
