diff --git a/packages/headless/src/modules/affiliate/useAffiliateLinkVisit.ts b/packages/headless/src/modules/affiliate/useAffiliateLinkVisit.ts
--- a/packages/headless/src/modules/affiliate/useAffiliateLinkVisit.ts
+++ b/packages/headless/src/modules/affiliate/useAffiliateLinkVisit.ts
@@ -90,7 +90,7 @@
 
     if (!response) {
       const origin = window.location.origin;
-      hasFailed.value = true;
+      hasFailed.value = false;
       lastResponse.value = { target: origin };
       return origin;
     }
