diff --git a/packages/headless/src/modules/affiliate/affiliate.schemas.ts b/packages/headless/src/modules/affiliate/affiliate.schemas.ts
index 4edce263ab..69a26eb52b 100644
--- a/packages/headless/src/modules/affiliate/affiliate.schemas.ts
+++ b/packages/headless/src/modules/affiliate/affiliate.schemas.ts
@@ -274,7 +274,7 @@ export function useCommissionsQuerySchema(): JsonSchema7 {
           additionalProperties: false,
           required: ["field", "dir"],
           properties: {
-            field: { enum: ["amount", "created_at"] },
+            field: { enum: ["created_at"] },
             dir: { enum: [SortDirection.ASC, SortDirection.DESC] }
           }
         }
