diff --git a/packages/headless/src/modules/affiliate/useClientAffiliate.ts b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
--- a/packages/headless/src/modules/affiliate/useClientAffiliate.ts
+++ b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
@@ -236,18 +236,9 @@
     return enrolAffiliate(id)
       .then(() => {
         writeError.value = undefined;
-        return Promise.all([
-          invalidateQueryByKey([...AFFILIATE_ACCOUNT_QUERY_KEY, id], {
-            exact: false
-          })(undefined),
-          invalidateQueryByKey([...AFFILIATE_BALANCE_QUERY_KEY, id], {
-            exact: false
-          })(undefined),
-          loadAreaSettings(resolvedBrandId.value).then(
-            v => (areaSettings.value = v),
-            () => (areaSettings.value = {})
-          )
-        ]);
+        return invalidateQueryByKey([...AFFILIATE_ACCOUNT_QUERY_KEY, id], {
+          exact: false
+        })(undefined);
       })
       .then(
         () => undefined,
