diff --git a/packages/headless/src/modules/affiliate/useClientAffiliate.ts b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
--- a/packages/headless/src/modules/affiliate/useClientAffiliate.ts
+++ b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
@@ -231,7 +231,7 @@
 
   async function enrol(): Promise<void> {
     const id = accountId.value;
-    if (!id || !isClientActor.value || isEnrolled.value) return;
+    if (!id || !isClientActor.value) return;
     isProcessing.value = true;
     return enrolAffiliate(id)
       .then(() => {
