diff --git a/packages/headless/src/modules/affiliate/useClientAffiliate.ts b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
--- a/packages/headless/src/modules/affiliate/useClientAffiliate.ts
+++ b/packages/headless/src/modules/affiliate/useClientAffiliate.ts
@@ -253,6 +253,7 @@
         () => undefined,
         err => {
           writeError.value = mapToHeadlessError(err);
+          return refresh();
         }
       )
       .finally(() => {
