diff --git a/packages/headless/src/modules/affiliate/useAffiliateLinks.actions.ts b/packages/headless/src/modules/affiliate/useAffiliateLinks.actions.ts
--- a/packages/headless/src/modules/affiliate/useAffiliateLinks.actions.ts
+++ b/packages/headless/src/modules/affiliate/useAffiliateLinks.actions.ts
@@ -81,6 +81,11 @@
   async function remove(linkId: string): Promise<void> {
     const accountId = keyAccountId.value;
     if (!accountId) throw new NotAuthenticatedError();
+    query.setCriteria({
+      filters: {},
+      sort: undefined,
+      pagination: undefined
+    });
     return removeLink(accountId, linkId).then(
       () => {
         writeError.value = undefined;
