diff --git a/packages/headless/src/modules/affiliate/affiliate.schemas.ts b/packages/headless/src/modules/affiliate/affiliate.schemas.ts
index 153344b182..ef7a40ef70 100644
--- a/packages/headless/src/modules/affiliate/affiliate.schemas.ts
+++ b/packages/headless/src/modules/affiliate/affiliate.schemas.ts
@@ -218,7 +218,19 @@ export function useReferralsQuerySchema(): JsonSchema7 {
           }
         }
       },
-      pagination: paginationSchema(5)
+      pagination: {
+        type: "object",
+        additionalProperties: false,
+        properties: {
+          limit: { type: "integer", minimum: 0, default: 5 },
+          offset: {
+            type: "integer",
+            minimum: 0,
+            maximum: 0,
+            default: PAGINATION.offset
+          }
+        }
+      }
     }
   } satisfies JsonSchema7;
 }
