diff --git a/packages/headless/src/modules/affiliate/useAffiliatePayoutDestinationManager.actions.ts b/packages/headless/src/modules/affiliate/useAffiliatePayoutDestinationManager.actions.ts
--- a/packages/headless/src/modules/affiliate/useAffiliatePayoutDestinationManager.actions.ts
+++ b/packages/headless/src/modules/affiliate/useAffiliatePayoutDestinationManager.actions.ts
@@ -54,7 +54,7 @@ export function createAffiliatePayoutDestinationManagerActions(
       .catch(error =>
         Promise.reject(
           new DetailedError(
-            t("error.affiliate_payout_destination_form_timeout"),
+            t("affiliate_payout_destination_form_timeout"),
             responseCodes.Timeout,
             ErrorOrigin.Headless,
             { error, state: state.value }
