diff --git a/packages/headless/src/modules/affiliate/affiliate.utils.ts b/packages/headless/src/modules/affiliate/affiliate.utils.ts
index d5e7b056f2..0405039c52 100644
--- a/packages/headless/src/modules/affiliate/affiliate.utils.ts
+++ b/packages/headless/src/modules/affiliate/affiliate.utils.ts
@@ -111,9 +111,8 @@ export function isPaypalDestination(
   destinationId: string | null | undefined,
   destinations: IAffiliateBrandPayoutDestination[] = []
 ): boolean {
-  const target = destinationId
-    ? find(destinations, ["id", destinationId])
-    : find(destinations, "default");
+  if (!destinationId) return false;
+  const target = find(destinations, ["id", destinationId]);
   return target?.code === AffiliatePayoutDestinationCode.PAYPAL;
 }
 
